src/api/middlewares/verifyToken.js

```diff
@@ -0,0 +1,39 @@
+const createError = require("http-errors");
+const jwt = require("jsonwebtoken");
+const { JWT_SECRET } = require("../../config/index");
+
+function verifyToken(req, res, next) {
+  const authToken = req.headers.authorization;
+
+  if (!authToken) {
+    return next(createError(401));
+  }
+
+  if (authToken.split(" ")[0] !== "Bearer") {
+    return next(createError(401));
+  }
+
+  const accessToken = authToken.split(" ")[1];
+
+  const verifiedUserData = jwt.verify(
+    accessToken,
+    JWT_SECRET,
+    (error, payload) => {
+      if (error) {
+        return null;
+      }
```

Comment:
```suggestion
      if (error) return null;
```
p4 이렇게 하시면 다른 코드와 일관성이 유지될 것 같습니다!